src/extend-domino.js

```diff
@@ -0,0 +1,44 @@
+//
+// Strict mode disallows us to overwrite Document.prototype properties.
+// This file is to stay out of strict mode.
```

Comment:
I see. My understanding is that the code is supposed to support `document.createElement` for custom elements. An example use case is a custom element creating another custom element, and adding it to a part of the page that has already been traversed.